WalletLibrary/WalletLibrary/Protocols/RootOfTrust/IdentifierDocumentResolver.swift

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * Defines the behavior of resolving an Identifier Document.
+ */
+protocol IdentifierDocumentResolver
```

Comment:
Should this be `IdentifierDocumentResolving`?

Reply:
yeah I was going back and forth between Swift standards for protocol naming and what I think it should be named but yea I can change it to IdentifierDocumentResolving to adhere to Swift guidelines.